### ISSUE TITLE
FEATURE: Update Zeek oui.dat during container build

### ISSUE THREAD

**Issue by dougburks** [Security-Onion-Solutions/securityonion#9542]
https://github.com/Security-Onion-Solutions/securityonion-image/pull/304

**Comment by dougburks** [comment]
Tested and verified that oui.dat has been updated and increased in size from 912K to 1.1M.

Before:
```
sudo docker exec -it so-zeek ls -alh /opt/zeek/share/zeek/site/oui-logging/oui.dat
-rw-r--r-- 1 zeek zeek 912K Dec 12 19:37 /opt/zeek/share/zeek/site/oui-logging/oui.dat
```

After:
```
sudo docker exec -it so-zeek ls -alh /opt/zeek/share/zeek/site/oui-logging/oui.dat
-rw------- 1 zeek zeek 1.1M Jan 10 14:34 /opt/zeek/share/zeek/site/oui-logging/oui.dat
```

**Comment by lock-wire** [comment]
Merged PRs #306 and #307 to update oui.dat during build for 2.3.200 and 2.4 respectively.
